app/src/components/ViewEvent/ViewEvent.tsx

```diff
@@ -85,3 +92,16 @@ const DateSection = ({ startDate, endDate }: IDateProps) => {
 
 export const capitalize = (text: string) =>
   text.charAt(0).toUpperCase() + text.substring(1);
+
+const DateTimeSection = (date: Date) => {
+  return (
+    <>
+      <p className={style.infoHeader}>Påmelding åpner</p>
+      <p className={style.dateText}>
+        {capitalize(format(date, 'cccc dd. MMMM yyyy', { locale: nb }))} <br />
+        Klokken:
+        {stringifyTime({ hour: date.getHours(), minute: date.getMinutes() })}
```

Comment:
Denne også, fint om vi definerer desse funksjonene ein plass.

Reply:
`stringifyTime` er allerede definert i `time.ts` men jeg tar en liten konvertering fra `Date` til `{hour: xx, minute: xx}` for å kunne bruke samme metode her. Bør jeg heller lage en ny `stringtfyTime` i `time-instance`?  Eller lage en funksjon som konverterer for meg så det slipper å være så 'dirty' kode her? 🤔

Reply:
Tror jeg fant en grei løsning på det :)